### ISSUE TITLE
Images disappear until the browser window size changes.

### ISSUE THREAD

**Issue by BeholdPL** [ApoorvSaxena/lozad.js#172]
Images disappear until the browser window size changes.

See the video below:
https://www.dropbox.com/s/4w6cuq4xvdwriz2/lozad.mov?dl=0

right now my site is on my local, in afternoon I move it to server, but until that - how could I fix it? Console is clear.

**Comment by alamowo** [comment]
This seems to have changed in Chrome v74/75. Not sure yet. Since that it's not working anymore.

**Comment by justinasdmt** [comment]
Hey,
I have the same problem. Chrome v75 doesn't load some images. Does anyone have a solution?

**Comment by justinasdmt** [comment]
I noticed that in my case images were not loading if their parent or grandparent element was set to overflow: hidden. I deleted this CSS property and lazy-loading works again.

**Comment by alamowo** [comment]
@justinasdmt this is working for me, too. The Issue seems to be related to the IntersectionObserver. It doesn't recognize elements that do not have any dimensions: 0px x 0px. For me adding a min-width or min-height of 1px did the trick.

**Comment by BeholdPL** [comment]
Yes. The problem is overflow: hidden on parent or grandparent element, like @justinasdmt wrote. @ApoorvSaxena is this lozad.js bug, or IntersectionObserver? :)

**Comment by BeholdPL** [comment]
@ApoorvSaxena do ho have any idea how to fix that issue? I really like your lozad, I would like to not change it to other...

**Comment by Gouigouix** [comment]
There is always the problem with Chrome:
- Version 76.0.3809.132 (Official Build) (64-bit)
- Version 78.0.3901.0 (Official Build) canary (64-bit)

A solution ?

**Comment by BeholdPL** [comment]
> There is always the problem with Chrome:
> 
> * Version 76.0.3809.132 (Official Build) (64-bit)
> * Version 78.0.3901.0 (Official Build) canary (64-bit)
> 
> A solution ?

Did you find any way to fix that maybe?

**Comment by man-oi** [comment]
same here. but _some_ images were loaded.
I was able to fix the error by reacting to the "loaded" event and checking which images are below or above the loaded image and then loading them manually.

this is only a quick fix and not perfect, but it works for now.

```javascript
let yOffset = window.pageYOffset;
let dir = 'down';
const observer = window.lozad('img.lozad', {
	loaded: (loadedImage) => {
	  if (!loadedTimeout) {
		loadedTimeout = 1;
		if (window.pageYOffset >= yOffset) dir = 'down';
		else dir = 'up';
		yOffset = window.pageYOffset;
		const loadedImageY = loadedImage.getBoundingClientRect().top - window.pageYOffset;
		document.querySelectorAll('img.lozad').forEach((el) => {
		  const elYPos = el.getBoundingClientRect().top - window.pageYOffset;
		  const threshold = window.innerHeight / 2;
		  if (dir === 'down') {
			if (elYPos <= loadedImageY + threshold) {
			  observer.triggerLoad(el);
			}
		  } else if (elYPos >= loadedImageY - threshold) {
			observer.triggerLoad(el);
		  }
		});
		window.setTimeout(() => { loadedTimeout = null; }, 400);
	  }
	},
});
observer.observe();
```

**Comment by man-oi** [comment]
now discovered instead of the above event function, it also works with:

```css
.lozad {
  min-width: 1px;
  min-height: 1px;
}
```

🙄
